### ISSUE TITLE
Community Control Nominations

### ISSUE THREAD

**Issue by adammatlack** [ioncoincore/ion#3]
Here is where i would like people to nominate and or second people/users to control host the ioncore repo and also its respective domain and website.

**Comment by adammatlack** [comment]
I will go first, I nominate two individuals, Korvas128, and Psionic.

**Comment by RdeWilde** [comment]
dkbay, sleak, Mitchellcash, ckti, tor, huey...

**Comment by JSponaugle** [comment]
Psionic, Korvas128 or Mitchellcash

**Comment by Blackwolfen** [comment]
lets solve the issue with a pullrequest and give a +1 vote for psionic.

**Comment by cryptolinux** [comment]
psionic was involved in masternode decision which was one of the triggers for current transition process, by that, no clearly.
Mitchellcash is part of ionomy.
Korvas128 had no trust issues and is not involved with ionomy and clearly ok.

**Comment by jpass022** [comment]
> psionic was involved in masternode decision which was one of the triggers for current transition process, by that, no clearly.
> Mitchellcash is part of ionomy.
> Korvas128 had no trust issues and is not involved with ionomy and clearly ok.

Now who's the liar cryptolinux(Tor), subverting this community vote and erasing RdeWilde's comment. It's never been about the community for you, it's only about you trying to take over Ion Core and trying to extort resources from Ionomy gfor your own gain.

**Comment by RdeWilde** [comment]
dkbay, sleak, mitchellcash, psionic, ceforce, huey

**Comment by cryptolinux** [comment]
wtih such reply you disqualify yourself

**Comment by jpass022** [comment]
> wtih such reply you disqualify yourself

with your replies and actions you disqualify yourself from being here at all cryptolinux.

**Comment by cryptolinux** [comment]
@JSponaugle it would be more productive if you explain why you suggest somebody as well as why do you decline somebody. Thumbs up or thumbs down have no arguments.

**Comment by Cryptoformer** [comment]
Korvas128 is fine with me

**Comment by RdeWilde** [comment]
dkbay, sleak, mitchellcash, ckti, huey

**Comment by SL34K** [comment]
+1 vote for Korvas128
I'm unsure about others nominated due to potential ties with ionomy

**Comment by RdeWilde** [comment]
dkbay, sleak, mitchellcash, ckti, huey

**Comment by RdeWilde** [comment]
dkbay, sleak, mitchellcash, ckti, huey

**Comment by RdeWilde** [comment]
dkbay, sleak, mitchellcash, ckti, huey, psionic

**Comment by RdeWilde** [comment]
dkbay, sleak, mitchellcash, ckti, huey, psionic

**Comment by RdeWilde** [comment]
dkbay, sleak, mitchellcash, ckti, huey, psionic

**Comment by RdeWilde** [comment]
Tor, no need for this, get over it and DM me and get on with it.

dkbay, sleak, mitchellcash, ckti, huey, psionic

**Comment by ckti** [comment]
Korvas128 has been added to the list of write-enabled users

**Comment by adammatlack** [comment]
I think if he would be willing Lorddarkhelmet would also be an excellent community addition

**Comment by adammatlack** [comment]
Is there a reason for why all of these comments are being deleted?

**Comment by cryptolinux** [comment]
@adammatlack yes, spamming, [user was asked to stop](https://github.com/ioncoincore/ion/issues/8) but ignored it too.

**Comment by cryptolinux** [comment]
LordDarkHelmet proved himself as true ion supporter without any bias shown over a long period of time, he clearly could contribute a lot, even if just by expressing opinions, if he finds time to spend on improving this project in any way, I would would fully agree with it.
@adammatlack this issue was chaotic, first a [project](https://github.com/ioncoincore/ion/projects/1) should have been where everything is defined. Roles for which one should be nominated should be set too, as having write access as well as who at all requires it as well as what tasks this person/s would have should be defined.

**Comment by cryptolinux** [comment]
I will keep this issue open for now, but it should be actually closed wit a ref to: https://github.com/ioncoincore/ion/issues/10

**Comment by cryptolinux** [comment]
korvas128 is nominated for managing this project on github, all other nominations should take place after roles and tasks are known. Feel free to reopen this issue if somebody thinks that it should be reopened for some reason.

**Comment by korvasc** [comment]
Close but not quite complete

Project Manager          Myself

My job is to give structure and to also help facilitate a smooth transition. I'm a jack for all trades master of none. I am here to assist both  Development & Finance    

Development               ckti 

Even though ckti is paid for by Ionomy he is also well respected by the community, I see no conflict of Interests.

Finance                         Propose **LordDarkHelmet**   

Again a well-respected long time member of this community with the expertise to be in a position to advise on the communities financial position.
